docs/blog/posts/2025/09/eessi-cray-slingshot11.md

```diff
@@ -0,0 +1,332 @@
+---
+author: [Richard]
+date: 2025-09-10 
+slug: EESSI-on-Cray-Slingshot
+---
+
+# MPI at Warp Speed: EESSI Meets Slingshot-11
+
+High-performance computing environments are constantly evolving, and keeping pace with the latest interconnect technologies is crucial for maximising application performance. However, we cannot rebuild all the software in EESSI that depends on improvements to communication libraries. So how do we take advantage of new technological developments?
+
+Specifically we look at taking benefit of the HPE/Cray Slingshot-11.
+Slingshot-11 promises to offer a significant advancement in HPC networking, offering improved bandwidth, lower latency, and better scalability for exascale computing workloads ... so
+this should be worth the effort!
+
+In this blog post, we present the requirements for building OpenMPI 5.x with Slingshot-11 support on HPE/Cray systems and its integration with EESSI using the [host_injections](../../../../site_specific_config/host_injections.md)
+mechanism of EESSI to inject custom-built OpenMPI libraries. This approach enables overriding EESSI’s default MPI library with an ABI-compatible, Slingshot-optimized version which should give us optimal performance.
+<!-- more -->
+
+## The Challenge
+
+EESSI provides a comprehensive software stack, but specialized interconnect support like Slingshot-11 can sometimes require custom-built libraries that aren't yet available in the standard EESSI distribution. Our goal is to:
+
+1. Build OpenMPI 5.x with native Slingshot-11 support
+2. Create ABI-compatible replacements for EESSI's OpenMPI libraries
+3. Place the libraries somewhere where EESSI automatically picks them up
+4. Support both x86_64 AMD CPU partitions and NVIDIA Grace CPU partitions with Hopper accelerators
+
+The main task is to build the required dependencies on top of EESSI, since many of the libraries needed for libfabric with CXI support are not yet available in the current EESSI stack.
+
+### System Architecture
+
+Our target system is [Olivia](https://documentation.sigma2.no/olivia_pilot_period_docs/olivia_pilot_main.html) which is based on HPE Cray EX platforms for compute and accelerator nodes, and HPE Cray ClusterStor for global storage, all
```

Comment:
I could imagine that the pilot page will be removed at some point. Maybe point to a different one?